BESMod/Resources/ReferenceResults/Dymola/BESMod_Examples_HighOrderModel_BES_HOM.txt

```diff
@@ -1,20 +1,20 @@
-last-generated=2024-11-08
+last-generated=2024-11-15
 statistics-initialization=
```

Comment:
Maybe, while at it, we could plot two zone temperatures instead of the electrical values. This way, the error would have been caught earlier, I think. Can you modify the simulate and plot script?

Reply:
We already plot two zone temperatures in one plot (zone 1 and zone 10). Or we increase the simulation time a little, e.g., from 2 to 3 days?